GSoC 2023 blog post draft

orbeckst opened the pull request:
Have a blog post lined up in case MDAnalysis is announced as a GSOC 2023 org on 2023-02-22.

orbeckst commented on _posts/2023-02-22-gsoc2023.md:
```diff
@@ -0,0 +1,127 @@
+|---------|-----------------------------------------------------------------------------------------------------------------------------------------------------------|------------|------------|------------------------------------------------------------------------------------------------|-----------------------------|----------------------------------------|
+| 1       | [Generalise Groups]({{ site.github.wiki }}/GSoC-2023-Project-Ideas#project-1-bead-and-ring-groups)                                         | hard     | 350 hours   | Generalise concept of groups                                                                   | Python, NetworkX, Molecular modelling                  | @lilyminium, @fiona-naughton, @richardjgowers, @IAlibay, @micaela-m ...
+| 2       | [Extend MDAnalysis Interoperability]({{ site.github.wiki }}/GSoC-2023-Project-Ideas#project-2-extend-interoperability)                  | medium     | 350 hours   | Extend converters module to other relevant packages                                            | Python, Molecular Modelling                      | @lilyminium, @IAlibay, @fiona-naughton, @hmacdope, @yuxuanzhuang, @RMeli|
+| 3       | [Molecular volume and surface analysis]({{ site.github.wiki }}/GSoC-2023-Project-Ideas#project-3-molecular-volume-and-surface-analysis) | medium       | 175 hours | use an existing package for molecular surface area calculations to build a new analysis module | Python, MDAnalysis.analysis, Molecular Modelling | @orbeckst, @IAlibay, @hmacdope, @yuxuanzhuang, @RMeli |
```
Probably remove from here and the ideas list because there's an open PR.

orbeckst commented on _posts/2023-02-22-gsoc2023.md:
```diff
@@ -0,0 +1,127 @@
+Introduce yourself on the [mailing list]. Tell us your github handle, what you plan to work
+on during the summer or what you have already done with MDAnalysis.
+
+### Close an issue of MDAnalysis
```
Do we need to be more specific here?

Need to check that blog post and FAQ are consistent.

orbeckst reviewed (commented):
Please look at the highlighted points during review. Thanks.

orbeckst commented on _posts/2023-02-22-gsoc2023.md:
```diff
@@ -0,0 +1,127 @@
+— MDAnalysis GSoC mentors
+
+
+[org]: https://summerofcode.withgoogle.com/programs/2023/organizations/mdanalysis
```
**MUST CHECK URL**

orbeckst replied:
URL works ;-)

RMeli commented on _posts/2023-02-22-gsoc2023.md:
```diff
@@ -0,0 +1,127 @@
+check the [ideas][] page — we might add more ideas after the posting date of
+this post.
+
+Our experience shows that having the listed skills increases the
+chances that a project will be completed successfully, so we use them
+as part of our decision criteria in choosing GSoC contributors.
```
I think this is very well phrased. Makes it clear what we look for in candidates without discouraging potential applicants.

RMeli commented on _posts/2023-02-22-gsoc2023.md:
```diff
@@ -0,0 +1,127 @@
+</p>
+
+MDAnalysis has been accepted as an [organization][org] for [Google Summer of
+Code 2023][gsoc]!  If you are interested in working with us this summer and you
```
```suggestion
Code 2023][gsoc]! If you are interested in working with us this summer and you
```

RMeli commented on _posts/2023-02-22-gsoc2023.md:
```diff
@@ -0,0 +1,127 @@
+MDAnalysis has been accepted as an [organization][org] for [Google Summer of
+Code 2023][gsoc]!  If you are interested in working with us this summer and you
+are new to open source, please read the advice and links below and write to us
+on the [mailing list].
```
```suggestion
on the [GSoC with MDAnalysis mailing list].
```
More specific and consistent with what's written below.

RMeli commented on _posts/2023-02-22-gsoc2023.md:
```diff
@@ -0,0 +1,127 @@
+are scoped as either 175-hour (medium) or 350-hour (long) size. The
+duration can be extended from the standard 12 weeks to 22 weeks.
+
+The application window **deadline** is **April 4, 2023 - 18:00 UTC
+18:00 (UTC)**. As part of the application process you must familiarize
```
```suggestion
The application window **deadline** is **April 4, 2023 - 18:00 UTC**.
As part of the application process you must familiarize
```

RMeli commented on _posts/2023-02-22-gsoc2023.md:
```diff
@@ -0,0 +1,127 @@
+
+As a start to get familiar with MDAnalysis and open source development you
+should follow these steps:
+
```
Add YouTube trailer? It's very short but might give candidates a good overview of what's MDAnalysis is about, if they don't know already.

```suggestion

### Watch the MDAnalysis Trailer

The [MDAnalysis Trailer](https://www.youtube.com/watch?v=uMAfvwFkD3o) on YouTube
is a one minute introduction to MDAnalysis.

```

RMeli commented on _posts/2023-02-22-gsoc2023.md:
```diff
@@ -0,0 +1,127 @@
+
+### Introduce yourself to us
+
+Introduce yourself on the [mailing list]. Tell us your github handle, what you plan to work
```
```suggestion
Introduce yourself on the [mailing list]. Tell us your GitHub handle, what you plan to work
```

RMeli commented on _posts/2023-02-22-gsoc2023.md:
```diff
@@ -0,0 +1,127 @@
+
+To start developing for MDAnalysis have a look at our
+guide on [contributing to MDAnalysis][dev-guide] and write to us on
+the [mailing list][] if you have more questions about setting up a
```
Which mailing list should this be? Still the GSoC one or the Developer one?

orbeckst replied:
I switched it from devs to gsoc to make sure that gsoc candidates only need to come to one place.

RMeli replied:
Ok, I just wanted to double check this was intended.

orbeckst commented:
Please have a quick check so that we can post this soon.

"LGTM" reviews are fine, this means some else except I and @RMeli read it. 

For "changes requested" please be specific or make the changes yourself.

lilyminium commented on _posts/2023-02-22-gsoc2023.md:
```diff
@@ -0,0 +1,148 @@
+
+| project | name                                                                                                                                                      | difficulty | project size | description                                                                                    | skills                      | mentors                                |
+|---------|-----------------------------------------------------------------------------------------------------------------------------------------------------------|------------|------------|------------------------------------------------------------------------------------------------|-----------------------------|----------------------------------------|
+| 1       | [Generalise Groups]({{ site.github.wiki }}/GSoC-2023-Project-Ideas#project-1-bead-and-ring-groups)                                         | hard     | 350 hours   | Generalise concept of groups                                                                   | Python, NetworkX, Molecular modelling                  | @lilyminium, @fiona-naughton, @richardjgowers, @IAlibay, @micaela-m ...
```
```suggestion
| 1       | [Generalise Groups]({{ site.github.wiki }}/GSoC-2023-Project-Ideas#project-1-bead-and-ring-groups)                                         | hard     | 350 hours   | Generalise concept of groups                                                                   | Python, NetworkX, Molecular modelling                  | @fiona-naughton, @richardjgowers, @yuxuanzhuang, @RMeli |
```

lilyminium commented on _posts/2023-02-22-gsoc2023.md:
```diff
@@ -0,0 +1,148 @@
+| project | name                                                                                                                                                      | difficulty | project size | description                                                                                    | skills                      | mentors                                |
+|---------|-----------------------------------------------------------------------------------------------------------------------------------------------------------|------------|------------|------------------------------------------------------------------------------------------------|-----------------------------|----------------------------------------|
+| 1       | [Generalise Groups]({{ site.github.wiki }}/GSoC-2023-Project-Ideas#project-1-bead-and-ring-groups)                                         | hard     | 350 hours   | Generalise concept of groups                                                                   | Python, NetworkX, Molecular modelling                  | @lilyminium, @fiona-naughton, @richardjgowers, @IAlibay, @micaela-m ...
+| 2       | [Extend MDAnalysis Interoperability]({{ site.github.wiki }}/GSoC-2023-Project-Ideas#project-2-extend-interoperability)                  | medium     | 350 hours   | Extend converters module to other relevant packages                                            | Python, Molecular Modelling                      | @lilyminium, @IAlibay, @fiona-naughton, @hmacdope, @yuxuanzhuang, @RMeli|
```
```suggestion
| 2       | [Extend MDAnalysis Interoperability]({{ site.github.wiki }}/GSoC-2023-Project-Ideas#project-2-extend-interoperability)                  | medium     | 350 hours   | Extend converters module to other relevant packages                                            | Python, Molecular Modelling                      | @fiona-naughton, @hmacdope, @yuxuanzhuang, @RMeli|
```

lilyminium commented on _posts/2023-02-22-gsoc2023.md:
```diff
@@ -0,0 +1,148 @@
+|---------|-----------------------------------------------------------------------------------------------------------------------------------------------------------|------------|------------|------------------------------------------------------------------------------------------------|-----------------------------|----------------------------------------|
+| 1       | [Generalise Groups]({{ site.github.wiki }}/GSoC-2023-Project-Ideas#project-1-bead-and-ring-groups)                                         | hard     | 350 hours   | Generalise concept of groups                                                                   | Python, NetworkX, Molecular modelling                  | @lilyminium, @fiona-naughton, @richardjgowers, @IAlibay, @micaela-m ...
+| 2       | [Extend MDAnalysis Interoperability]({{ site.github.wiki }}/GSoC-2023-Project-Ideas#project-2-extend-interoperability)                  | medium     | 350 hours   | Extend converters module to other relevant packages                                            | Python, Molecular Modelling                      | @lilyminium, @IAlibay, @fiona-naughton, @hmacdope, @yuxuanzhuang, @RMeli|
+| 3       | [Benchmarking and performance optimization]({{ site.github.wiki }}/GSoC-2023-Project-Ideas#project-3-benchmarking-and-performance-optimization)                  | medium/hard     | 175 hours   |  write benchmarks for automated performance analysis and address performance bottlenecks | Python, Molecular Modelling                      | @hmacdope, @orbeckst, @ojeda-e, @RMeli |
```
```suggestion
| 3       | [Benchmarking and performance optimization]({{ site.github.wiki }}/GSoC-2023-Project-Ideas#project-3-benchmarking-and-performance-optimization)                  | medium/hard     | 175 hours   |  write benchmarks for automated performance analysis and address performance bottlenecks | Python, Molecular Modelling                      | @hmacdope, @orbeckst, @RMeli |
```

lilyminium reviewed (changes requested):
I updated the table of mentors here and in the wiki to remove those who are not available to formally mentor this year. Otherwise LGTM!

IAlibay reviewed (approved):
LGTM

I'm probably going to be involved with some of these projects if they go ahead, but yeah I'd prefer stay as an admin on paper, thanks for remembering @lilyminium .

hmacdope reviewed (approved):
LGTM

orbeckst commented:
- [x] updated https://github.com/MDAnalysis/mdanalysis/wiki/Google-Summer-Of-Code
- [x] updated https://github.com/MDAnalysis/mdanalysis/wiki/GSoC-FAQ
- [x] announced by Google https://summerofcode.withgoogle.com/programs/2023/organizations

Ready to merge.
